### ISSUE TITLE
The DryDep collection is not activated for GCHP fullchem benchmarks

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#2139]
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/CONTRIBUTING.html)

### New GEOS-Chem feature or discussion

The DryDep diagnostic is commented out by default in the GCHP fullchem benchmark simulation.  ~~I believe this had been done due to avoid an ESMF "Input file too long" error.~~ However, we need the DryDep diagnostic output to plot the Ox budget and drydep velocities.

Potential fix: Activate the DryDep diagnostic, but only archive the fields that are needed.

Tagging @msulprizio @lizziel

**Comment by yantosca** [comment]
Activating the DryDep collection (with other collections) caused the problem in #2135.  I am testing if just activating the DryDep collection will allow GCHP benchmark simulations to run.

**Comment by yantosca** [comment]
Using a reduced definition for the DryDep collection gets past the error.  I think this is due to some leftover species which may have been removed in the GCHP HISTORY.rc file.  Using this definition for the DryDep collection works:
```console
```console
#==============================================================================
# %%%%% THE DryDep COLLECTION %%%%%
#
# Dry depositon fluxes and velocities (for all dry-depositing species)
#
# Available for all simuations that have dry-depositing species
#==============================================================================
  DryDeptemplate:        '%y4%m2%d2_%h2%n2z.nc4',
  DryDep.format:          'CFIO',
  DryDep.timestampStart:  .true.
  DryDep.monthly:         0
  DryDep.frequency:       010000
  DryDep.duration:        010000
  DryDep.mode:            'time-averaged'
  DryDep.fields:          'DryDepVel_ACET       ', 'GCHPchem',
                          'DryDepVel_HNO3       ', 'GCHPchem',
	                  'DryDepVel_NH3        ', 'GCHPchem',
                          'DryDepVel_NH4        ', 'GCHPchem',
                          'DryDepVel_NIT        ', 'GCHPchem',
                          'DryDepVel_NITs       ', 'GCHPchem',
                          'DryDepVel_O3         ', 'GCHPchem',
                          'DryDepVel_SO4        ', 'GCHPchem',
                          'DryDep_O3            ', 'GCHPchem',
::
```

**Comment by lizziel** [comment]
I can try to improve the error handling for this type of problem.

**Comment by yantosca** [comment]
@lizziel: I've opened draft PR #2144 where I will test this.  This can go into 14.4.0.

**Comment by yantosca** [comment]
PR #2289 has been merged into GEOS-Chem 14.4.0.  We can now close out this issue.
